keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -407,18 +568,24 @@ Due to change of Kubernetes behavior, we will implement the feature only for cas
 #### Phase 1
 - Implement the feature only for volumes that are backed by PersistentVolumeClaims with `ReadWriteOncePod` access mode.
   Such volumes can be used only in a single pod and two pods can't ever conflict when using it.
-- Collect metrics of how many other pods would fail because they use a RWO/RWX volume that's used by a pod with different SELinux context on the same node.
-  TBD: create Info level alert ("please consider re-architecting your app not to share volumes this way and/or report it to sig-storage")?
-This phase can go Beta (be enabled by default) or even GA without breaking anything.
+- Collect metrics of how many other pods would fail because they use a RWO/RWX volume that's used by a pod with different SELinux label on the same node.
+
+This phase went Beta (enabled by default) in Kubernetes 1.28, without `SELinuxChangePolicy` field in PodSpec!
+
+#### Phase 1.5
```

Comment:
Phase 1.5 seems like a cop out. We implementing this really in 3 phases, there are no questions about it IMO.

Reply:
I added explicit phase 2 and 3 as a separate commit. I wanted the phases to be comparable with a previous version, but I agree that 1.5 is ugly.